Ginger/Ginger/ALM/QC/TreeViewItems/QCTestSetTreeItem.cs

```diff
@@ -96,7 +96,8 @@ public void IsTestSetAlreadyImported()
                     {
                         AlreadyImported = true;
                         MappedBusinessFlow = bf;
-                        MappedBusinessFlowPath = System.IO.Path.Combine(bf.ContainingFolder, bf.Name);                       
+                        string businessFlowName = Amdocs.Ginger.Common.GeneralLib.General.RemoveInvalidFileNameChars(bf.Name);
```

Comment:
why we removing invalid chars from here ? i think its already removed when saving file.